src/EditCommand.php

```diff
@@ -5,6 +5,7 @@
 use Symfony\Component\Process\Process;
 use Symfony\Component\Console\Command\Command;
 use Symfony\Component\Console\Input\InputInterface;
```

Comment:
Ensure your use statements are shortest to longest top to bottom. (like they were before you added InputOption